pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -132,7 +132,7 @@ func (e esGateway) esGatewayRole() *rbacv1.Role {
 				APIGroups:     []string{""},
 				Resources:     []string{"secrets"},
 				ResourceNames: []string{},
-				Verbs:         []string{"get"},
+				Verbs:         []string{"get", "list", "watch"},
```

Comment:
ES Gateway now requires both "list" and "watch" privileges. It does a "list" when it initially boots up to load all the relevant secrets into memory (cache). 

Then it uses "watch" to keep its cache of secrets up-to-date.